src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1999,6 +1999,24 @@ limitations under the License.
             <include cluster="Temperature Control" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="Operational State" client="false" server="true" clientLocked="true" serverLocked="true"></include>
         </clusters>
+    </deviceType>
+        <deviceType>
+        <name>MA-laundry-dryer</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Laundry Dryer</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x007C</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="On/Off" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Laundry Mode" client="false" server="false" clientLocked="true" serverLocked="false"></include>
```

Comment:
But ... there is no "Laundry Mode" cluster in Matter XML, and this does not match the current spec either.   @eahove